spring-core/src/test/java/facade/BookingFacadeImplTest.java

```diff
@@ -0,0 +1,154 @@
+package facade;
+
+import model.Event;
+import model.Ticket;
+import model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Objects;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(locations = "classpath:beans.xml")
```

Comment:
need to remove, this is not integration test, need remove context creation

Reply:
why not ?
https://docs.spring.io/spring-batch/docs/current/reference/html/testing.html#creatingUnitTestClass

Reply:
Because for integration you have different class with context, this is class with test cases, not needed instance here, you need just test logic

Reply:
https://blog.gurock.com/unit-testing-tdd-bdd/